dockerfiles/backfill_events/startup_script.sh

```diff
@@ -4,4 +4,4 @@
 # In other words, just after the block number where the new collateral(s) was added to execute.
 echo "Starting event backfill to block [0]"
```

Comment:
I missed this earlier, but it's nice to update this so that the end point shows up in the logs. Would be best to extract this to a variable and update in a single place.